src/App.css

```diff
@@ -1,39 +1,9 @@
-.App {
-  text-align: center;
-}
-
```

Comment:
**[OPTIONAL]**

- [ ] Kindly remove the Live Demo section from your readme file because you have no live link in your Live Demo section so kindly remove the Live Demo section to make your readme descriptive and professional 👍